packages/nuxt-styleguide/package.json

```diff
@@ -18,16 +18,16 @@
     "@sum.cumo/vue-markdown-component-loader": "^0.5.0",
```

Comment:
but this is a private dep, no? 😕 ❓

Reply:
No: https://www.npmjs.com/package/@sum.cumo/vue-markdown-component-loader